Project-Hotpot/Project-Hotpot/CreatePlaylistViewController.swift

```diff
@@ -17,9 +17,12 @@ class CreatePlaylistViewController: UIViewController {
     
     @IBAction func didTapCreate(_ sender: Any) {
         //TODO: create edge case checks: empty text fields, etc.
-        PFPlaylist.createPlaylistInBackground(user: PFUser.current()!, name: playlistNameField.text ?? "test123") {playlist in
+        guard let user = PFUser.current(),
+              let playlistName = playlistNameField.text
+        else { return }
```

Comment:
Looks like you need to fix formatting with indentation and else brackets

Reply:
Or maybe im wrong looks like this is in combination with the guard statement and is supposed to be formatted like this?

Reply:
Is putting things in one line like else { return } bad style? For better readability should I put the return on a separate lines?

Reply:
We tend to write this as:
```
guard let user = PFUser.current(),
      let playlistName = playlistNameField.text else {
    return
}
```